components/SignupPage.js

```diff
@@ -28,17 +28,30 @@ class SignupPage extends Component {
     input: PropTypes.object,
     handleSubmit: PropTypes.func,
     authenticateUserMutation: PropTypes.func,
+    signupUserMutation: PropTypes.func,
     navigation: PropTypes.object
   }
 
-  handlePress = (values) => {
-    this.props.signup(values).then((response) => {
+  handlePress = async (values) => {
+    const {
+      firstName, lastName, email, password, passwordConfirm, phoneNum
+    } = values;
+    console.log(email, password);
+    try {
+      const response = await this.props.signupUserMutation({
+        variables: {
+          firstName, lastName, email, password, passwordConfirm, phoneNum
+        }
+      });
       const { navigate } = this.props.navigation;
-      navigate('CreateFamilyPage');
-      console.log(response.data);
-    }).catch(err => console.log(err));
+      const tokenToString = response.data.signupUser.token.toString();
+      this._storeAuthTokensLocally(tokenToString);
+      // Did not destructure because the variable name firstName exists
+      navigate('CreateFamilyPage', { name: response.data.signupUser.firstName });
+    } catch (e) {
+      console.error('An error occurred: ', e);
+    }
   }
-
   _handleFacebookLogin = async () => {
```

Comment:
Does the underscore naming hold any particular significance? I ask because I'd like to know if this function is being mapped to some function in the graphcool server directory and so must bear the exact name that one bears.

Reply:
It holds no meaning. I also think taking out the underscore may be better, such that we have consistent function names. Thanks.